src/Presentation/Views/Account/Account.tsx

```diff
@@ -0,0 +1,11 @@
+
+function Account() {
```

Comment:
What is the "Account" view? 

On Android, we are probably using Login / Register/ ForgotPass / Home (or welcome) as our screen names.

Is this the view to edit an existing account? If so, and just a nit, maybe EditAccount would be a better name?